frontend/src/components/pod/Details.tsx

```diff
@@ -528,10 +528,23 @@ export default function PodDetails(props: PodDetailsProps) {
             name: t('Host IP'),
             value: item.status.hostIP ?? '',
           },
+          {
+            name: t('Host IPs'),
+            value: item.status.hostIPs
+              ? item.status.hostIPs.map(ipObj => ipObj.ip).join(', ')
+              : '',
+            hideLabel: item.status.hostIPs?.length === 0,
+          },
           {
             name: t('Pod IP'),
             value: item.status.podIP ?? '',
           },
+          {
+            name: t('Pod IPs'),
+            value: item.status.podIPs
+              ? item.status.podIPs.map(ipObj => ipObj.ip).join(', ')
+              : ''
```

Comment:
this entry should also be hidden if there are no pod ips right?

Reply:
fixed